Formula/docker-credential-helper-ecr.rb

```diff
@@ -0,0 +1,25 @@
+class DockerCredentialHelperEcr < Formula
+  desc "Docker Credential Helper for ECR (Amazon EC2 Container Registry)"
+  homepage "https://github.com/awslabs/amazon-ecr-credential-helper"
+  url "https://github.com/awslabs/amazon-ecr-credential-helper/archive/e63ef99.tar.gz"
```

Comment:
This is going to need to be an actual release.

Reply:
FWIW Homebrew maintainers, this is getting parsed as version `99`. Smells a bit odd.